packages/integrations/src/adguard-home/adguard-home-integration.ts

```diff
@@ -124,6 +124,7 @@ export class AdGuardHomeIntegration extends Integration implements DnsHoleSummar
   }
 
   public async disableAsync(duration?: number): Promise<void> {
+    duration = duration ? duration * 1000 : 0;
```

Comment:
Small possible improvement, you can just set the default value of duration right in the function's parameter definition. Instead of "(duration?: number)", you can do "(duration = 0)".
You then won't need to test for undefined.
And instead of doing the *1000 here, apply it down in the body of the request.
(0*1000 is still 0 so it's fine)